force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -31,10 +31,16 @@ public with sharing class VariablesDataTypesOperators {
      * @param b The second number.
      * @return The sum of the two numbers, or null if either number is null.
      */
-    public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
-    }
-
+    public static Integer addition(Integer a, Integer b) { 
+        Integer sum;
+        if(a == null || b == null) {
+        return null;
+        }   
+        else {
+        sum = a + b;
+        return sum;
+        }
```

Comment:
Good work here. The logic is fine. What you might way to think about is if you can be more concise. That final else condition is really not needed. And typically I won't assign a value to variable if all I am doing is returning it the next line. 

```
	public static Integer addition(Integer a, Integer b) {
		if (a == null || b == null) {
			return null;
		}
		return a + b;
	}
	```